publish/dir.props

```diff
@@ -4,8 +4,13 @@
 
   <PropertyGroup>
     <ContainerName Condition="'$(ContainerName)' == ''">dotnet</ContainerName>
+    <ChecksumContainerName Condition="'$(ChecksumContainerName)' == ''">dotnet</ChecksumContainerName>
     <BaseUrl Condition="'$(BaseUrl)' == ''">https://dotnetcli.blob.core.windows.net/</BaseUrl>
     <NuGetPushTimeoutSeconds Condition="'$(NuGetPushTimeoutSeconds)' == ''">3600</NuGetPushTimeoutSeconds>
+    <BinariesRelativePath>$(Channel)/Binaries/$(SharedFrameworkNugetVersion)</BinariesRelativePath>
+    <InstallersRelativePath>$(Channel)/Installers/$(SharedFrameworkNugetVersion)</InstallersRelativePath>
+    <ChecksumsRelativePath>Runtime/Checksums/$(ProductVersion)</ChecksumsRelativePath>
```

Comment:
The Sdk checksums don't have a "Checksums" folder.  Let's keep it consistent between the Runtime and the SDK.  This should just be 

`<ChecksumsRelativePath>Runtime/$(ProductVersion)</ChecksumsRelativePath>`